.travis.yml

```diff
@@ -153,6 +158,10 @@ script:
   - share/spack/qa/run-$TEST_SUITE-tests
   - if [[ "$COVERAGE" == "true" ]]; then codecov --env PY_VERSION --required ; fi
 
+after_success:
+  - codecov --env PY_VERSION
```

Comment:
This line needs to be removed as it went in the `install` phase above after #6877 was merged (to ensure that Travis fails when data can't be uploaded).

Reply:
Good catch, I just merged it wrong...Done.